etcd/etcd.go

```diff
@@ -246,3 +236,29 @@ func (c *Client) Delete(id string) error {
 
 	return err
 }
+
+func (c *Client) UpsertAll(routes eskip.RouteList) error {
+	for _, r := range routes {
+		r.Id = routeid.GenerateIfNeeded(r.Id)
+		err := c.Upsert(r)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (c *Client) DeleteAllIf(routes eskip.RouteList, cond eskip.RoutePredicate) error {
```

Comment:
this function seems like a convenience function, and it is used only in the eskip tool. Why not put there where it is used?